freqtrade/configuration/cli_options.py

```diff
@@ -171,7 +169,7 @@ def __init__(self, *args, **kwargs):
         default=constants.HYPEROPT_EPOCH,
     ),
     "spaces": Arg(
-        '-s', '--spaces',
+        '--spaces',
```

Comment:
not really my contribution ...
it fails since they are all in the same argparse Namespace now - it's argparse who found this (did not let me pass ...)